### PR TITLE
Finished GenServer Cache

### PR DESCRIPTION
**Pull request by Alfedi**
Thanks to @Ironjanowar, I were able to finish this enhacement related to #6.
I use a GenServer to store the `file_id` variable so it's no necessary to upload the document to Telegram all the time.

### CODE REVIEW

File: lib/acm_bot.ex [hunk 1]
```diff
@@ -8,7 +8,8 @@ defmodule AcmBot do
 
     children = [
       ExGram,
-      {AcmBot.Bot, [method: :polling, token: token]}
+      {AcmBot.Bot, [method: :polling, token: token]},
+      AcmBot.Cache
```
Comment (rockneurotiko) [inline lib/acm_bot.ex]: I recommend to start the Cache before the Bot, in that case you won't have any problems like receiving an update with a file and trying to go to the Cache and that it doesn't exists

File: lib/acm_bot/cache.ex [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    GenServer.cast(__MODULE__, {:set_file_id, id})
+  end
+
+  def get_file_id() do
+    GenServer.call(__MODULE__, :get_file_id)
+  end
+
+  # Server Callbacks
+  def init(:ok) do
+    {:ok, ""}
```
Comment (rockneurotiko) [inline lib/acm_bot/cache.ex]: A cache that only saves an ID it's a little weird. What would happen if you need to save another file ID here? Would you create 2 AcmBot.Cache, one for every ID?

The typical way of doing this is having the inner state as a map, and accessing the cache with a key, for example:
`AcmBot.Cache.get(:file_id_comida)` and `AcmBot.Cache.put(:file_id_comida, file_id)`

**Review by rockneurotiko — CHANGES_REQUESTED** [review]
Comments about the Cache, I think that having a more generic cache is better than having only one that handles one string

**Review by Ironjanowar — CHANGES_REQUESTED** [review]
I think that the child starting order is important, didn't thing about that. Thanks @rockneurotiko

**Review by rockneurotiko — APPROVED** [review]
Mis 🔟 (Just change the children order before merging)

**Review by Ironjanowar — APPROVED** [review]

**Pull request merged**